app/src/main/java/com/futurice/freesound/feature/search/SoundItemViewHolder.java

```diff
@@ -80,12 +93,11 @@ public void unbind() {
     SoundItemViewHolder(@NonNull final View view,
                         @NonNull final Picasso picasso) {
         super(get(view));
-        this.rootView = view;
+        ButterKnife.bind(this, view);
+        rootView = view;
         this.picasso = get(picasso);
```

Comment:
I feel like we should decide about how to assign fields in constructors: I've typically written, `this.rootView = view` regardless of whether the field was named the same as the field. I think this makes for more consistent reading. Or at least we should change the name of the parameter so that this convention is more uniformly applied.